boards/adi/max32690fthr/Kconfig.max32690fthr

```diff
@@ -0,0 +1,7 @@
+# MAX32690 board configuration
+
+# Copyright (c) 2023-2024 Analog Devices, Inc.
```

Comment:
Is 2023-2024 intentional?

Reply:
Yes, I copy/pasted portions from the `max32690evkit`